docs/building.md

```diff
@@ -95,8 +95,8 @@ The following instructions assume that you are building `application-services` f
 
 1. Install Android SDK, JAVA, NDK and set required env vars
    1. Clone the [firefox-android](https://github.com/mozilla-mobile/firefox-android) repository (**not** inside the Application Service repository).
-   1. Install [Java **11**](https://www.oracle.com/java/technologies/downloads/#java11) for your system
-   1. Set `JAVA_HOME` to point to the JDK 11 installation directory.
+   1. Install [Java **17**](https://www.oracle.com/java/technologies/downloads/#java17) for your system
```

Comment:
Do app services developers also need to update? we can make a mini announcement in slack if so

Reply:
If they still have JDK11 installed, they'll need to update to 17, yeah. That said, they really *should* be on at least 17 at this point anyway! 😄